config/jobs/kubernetes/sig-testing/make-test.yaml

```diff
@@ -67,6 +67,38 @@ presubmits:
               cpu: 4
               memory: "36Gi"
 periodics:
+  - interval: 1h
+    name: ci-kubernetes-unit
+    annotations:
+      fork-per-release: "true"
+      testgrid-num-failures-to-alert: '3'
+      testgrid-alert-stale-results-hours: '24'
+      testgrid-create-test-group: 'true'
+      testgrid-dashboards: sig-release-master-informing
+    decorate: true
+    cluster: k8s-infra-prow-build
+    extra_refs:
+      - org: kubernetes
+        repo: kubernetes
+        base_ref: master
+        path_alias: k8s.io/kubernetes
+    spec:
+      containers:
+        - image: gcr.io/k8s-testimages/kubekins-e2e:v20210309-0688fde-master
+          command:
+            - make
+            - test
+            - KUBE_RACE=-race
+            # TODO: stop depending on custom timeout
+            - KUBE_TIMEOUT=--timeout=240s
+          # TODO: direct copy from pull-kubernetes-bazel-test, tune these
+          resources:
+            limits:
+              cpu: 4
+              memory: "36Gi"
+            requests:
+              cpu: 4
+              memory: "36Gi"
```

Comment:
wowza! 36 :)

Reply:
sooo... are these coming from what the bazel jobs used? or have we taken a look at resource consumption and this is still basically where we're at when trying to do as much in parallel as possible?

Reply:
> sooo... are these coming from what the bazel jobs used? or have we taken a look at resource consumption and this is still basically where we're at when trying to do as much in parallel as possible?

These are coming from the bazel jobs (see comment on line 95 above). We run CI on highmem nodes where 52gb : 8 cores is the ratio.

We can likely reduce this, and the timeout, and enable `-race` by default in `make test`, but it's more important to get it stable and blocking.

In 1.21 I will be tuning these, including a push to require unit tests to pass as a non-root user without privileges (in progress but probably landing in 1.21 realistically).

There's a lot wrong with how we've been running unit tests. Starting with lift and shift from bazel 🙃

Reply:
>it's more important to get it stable and blocking

completely agree, I don't think I phrased well... it was purely curiosity from my end, I didn't know if we'd bothered tuning anything yet (I know there's been a lot of work bringing individual test times down)

Reply:
just individual test packages, nothing else has received any tuning yet.
it's very difficult to safely tune resource usage as a prowjob author, we don't point you to anything to observe consumption, biasing on the side of asking for too much is safer.